dashboard/app/views/scripts/show.html.haml

```diff
@@ -60,7 +60,7 @@
         %br/
         %br/
 
-  - if can? :edit, @script
+  - if current_user.try(:levelbuilder?)
```

Comment:
thank you for catching these. it looks like there is one more to update here: https://github.com/code-dot-org/code-dot-org/blob/28112b9b0a4718242b32a9b1b78d95624ba47006/dashboard/app/views/courses/show.html.haml#L23-L24

Reply:
hi James, I saw you re-requested a review, but I didn't see any new commits or PR comments. I think this looks good to go once courses/show is no longer using `can?`.

Reply:
Oh sorry!  Weird -- when I checked earlier, I didn't see any reviews.  Must be some caching issue.  ;-)

Reply:
Nice catch, thanks!